pygmt/src/info.py

```diff
@@ -94,6 +94,10 @@ def info(table, **kwargs):
             # instead of a raw string that is less useful.
             if result.startswith(("-R", "-T")):  # e.g. -R0/1/2/3 or -T0/9/1
                 result = result[2:].replace("/", " ")
-            result = np.loadtxt(result.splitlines())
+            try:
+                result = np.loadtxt(result.splitlines())
+            except ValueError:
+                # Load non-numerical outputs in str type, e.g. for datetime
+                result = np.loadtxt(result.splitlines(), dtype="str")
```

Comment:
It means all the output columns will be in str type, including numerical columns. I'm not sure if I like it.

Reply:
Yes, but there's no way to have mixed types in numpy. An array with str (e.g. datetime) and numerical (e.g. int/float) types will have to be made into a `str` or `object` type.

Reply:
I'm OK with a string-type array. We can think more about the return values later (https://github.com/GenericMappingTools/pygmt/issues/593).